EventPlanning.service/Controllers/EventsController.cs

```diff
@@ -176,5 +176,11 @@ public IActionResult PatchEvent([FromBody]EventData eventData, [FromRoute]int ev
         {
             return eventManager.PatchEvent(eventData, eventId) ? StatusCode(200) : StatusCode(400);
         }
+
+        [HttpPut("{eventId}")]
+        public IActionResult AddActivitiesToEvent([FromBody]List<ActivityData> activityData, [FromRoute]int eventId)
```

Comment:
I would say that in front end it should be, whenever we click to add activity it will call it with one activity at a time not a list of activities.